### PR TITLE
Applicant - Delete account via Settings

### LINKED ISSUE #1144: Feature - Applicant/Manager(?) - Account deletion
# Description

Until we add a self-service account deletion feature, admins should be allowed to soft-delete users.

### PR DESCRIPTION
**Pull request by gggrant**
Resolves #1144.

### Notes:
- Basic users only.

- [x] Translations received

### CODE REVIEW

**Comment by gggrant** [comment]
```
'danger_zone' => 'Danger zone!',
'delete_alert_prefix' => 'Enter your email address',
'delete_alert_suffix' => 'below to confirm you want to permanently delete your Talent Cloud account.',
'confirm_delete_label' => 'Confirm account deletion',
'delete_placeholder' => 'Type your email address to activate the delete button.',
'delete_button' => 'Delete Account',
'delete_account' => 'Your account was deleted successfully.',
```

**Comment by mattejackson** [comment]
FR translation

'danger_zone' => ' Zone de danger!',
'delete_alert_prefix' => ' Entrez votre adresse de courriel',
'delete_alert_suffix' => ' ci dessous pour confirmer que vous souhaitez supprimer définitivement votre compte du Nuage de talents.',
'confirm_delete_label' => ' Confirmez la suppression du compte',
'delete_placeholder' => ' Saisissez votre adresse de courriel pour activer le bouton Supprimer.',
'delete_button' => ' Supprimez le compte',
'delete_account' => 'Votre compte a été supprimé avec succès.',

File: webpack.mix.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const mix = require("laravel-mix");
 const sass = require("node-sass");
 const cssnano = require("cssnano");
-const autoprefixer = require("autoprefixer");
+// const autoprefixer = require("autoprefixer");
```
Comment (Khristophor) [inline webpack.mix.js]: Any reason the autoprefixer was commented out?

File: package-lock.json [hunk 2]
```diff
@@ -2331,12 +2331,6 @@
                     "integrity": "sha1-jpQ9E1jcN1VQVOy+LtsFqhdO3hQ=",
```
Comment (Khristophor) [inline package-lock.json]: Not sure this should have been committed, I don't think any of the npm deps were changed as part of this PR.

File: webpack.mix.js (hunk 1 above)
Comment (gggrant) [reply webpack.mix.js]: I don't think it's being used. But I didn't mean to leave it like that. Undone.

File: package-lock.json (hunk 2 above)
Comment (gggrant) [reply package-lock.json]: True dat.

**Comment by Khristophor** [comment]
Chatted with @gobyrne about this on Friday, and there's a few things we should tackle first:

- Double check if instanced job application data holds recognizable user information (name, email, etc.)
- If instanced job application data does hold user information, add some sort of a disclaimer to the deletion area that for policy reasons we need to hold on to their information for 5 years (copy to come from Gray/product)
- "Scramble" identifiable user information when a user is soft deleted (use Faker or some other random string generation for name and email fields)
- Scrambling info on the user model will prevent the situation where a user deletes their account, comes back later to register, and is informed their email is taken

**Comment by gggrant** [comment]
- They do 👍 
- Forthcoming (copy)
- 'Deleted' for names and email rather than moving faker out of dev dependencies (?)
- 👍

**Comment by Khristophor** [comment]
>     * They do 👍
> 
>     * Forthcoming (copy)
> 
>     * 'Deleted' for names and email rather than moving faker out of dev dependencies (?)
> 
>     * 👍

Updating to `deleted` is probably fine

**Comment by mattejackson** [comment]
Delete Account Disclaimer copy (EN/FR) here: 

https://docs.google.com/document/d/11-_8ZiGNIa_ufg_lHbighUYSklynZRQAfzfC99gvkCc/edit

File: resources/lang/en/applicant/home.php [hunk 3]
```diff
@@ -216,5 +216,6 @@
     'team_button_twitter_label' => 'Twitter',
     'team_button_email_link' => 'talent.cloud-nuage.de.talents@tbs-sct.gc.ca',
     'team_button_email_title' => 'Get in touch with GC Talent Cloud via email.',
-    'team_button_email_label' => 'Email'
+    'team_button_email_label' => 'Email',
+    'success_title' => 'All good in the hood.'
```
Comment (Khristophor) [inline resources/lang/en/applicant/home.php:220]: Is this the official copy? 😆

**Pull request merged**